identity-iota/src/tangle/compression_brotli.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error::Error;
+use crate::error::Error::CompressionError;
+use std::io::prelude::*;
+
+pub(crate) fn compress_brotli<T: AsRef<[u8]>>(input: T) -> Result<Vec<u8>, Error> {
+  let mut result = Vec::new();
+  let mut compressor = brotli::CompressorReader::new(input.as_ref(), 4096, 5, 22);
```

Comment:
Same suggestion: please declare constants for magic numbers at the top of the file to know what they are.

E.g.
```rust
const BUFFER_SIZE: usize = 4096;
const QUALITY: u32 = 5; // compression level
const WINDOWS_SIZE: u32 = 22; // LZ77 window size
```